AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/InitialSync/InitialSyncOperation.swift

```diff
@@ -52,6 +59,7 @@ final class InitialSyncOperation: AsynchronousOperation {
 
         log.info("Beginning sync for \(modelType.modelName)")
         let lastSyncTime = getLastSyncTime()
+        initialSyncOperationTopic.send(.isFullSync(modelType, lastSyncTime == nil))
```

Comment:
Need to send two info (`modelType`: which model to be synced, and `lastSyncTime`: to determine the `syncType` of this `modelType`) to subscriber: `ModelSyncedEventEmitter`